src/errors.rs

```diff
@@ -4,7 +4,6 @@ use base64;
 use http;
 use regex;
 use reqwest;
-use serde_json;
```

Comment:
Thanks for cleaning this up. I think you can drop the other imports too.

Reply:
done